src/transformers/generation_tf_utils.py

```diff
@@ -479,21 +478,23 @@ def generate(
             use_cache: (`bool`, *optional*, defaults to `True`):
                 Whether or not the model should use the past last key/values attentions (if applicable to the model) to
                 speed up decoding.
-            output_attentions (`bool`, *optional*, defaults to `False`):
-                Whether or not to return the attentions tensors of all attention layers. See `attentions` under
-                returned tensors for more details.
-            output_hidden_states (`bool`, *optional*, defaults to `False`):
-                Whether or not to return the hidden states of all layers. See `hidden_states` under returned tensors
+            output_attentions (`bool`, *optional*, defaults to `model.config.output_attentions` or `False` if the
+                config does not set any value): Whether or not to return the attentions tensors of all attention
+                layers. See `attentions` under returned tensors for more details.
+            output_hidden_states (`bool`, *optional*, defaults to `model.config.output_hidden_states` or `False` if the:
+                config does not set any value): Whether or not to return the hidden states of all layers. See
+                `hidden_states` under returned tensors for more details.
+            output_scores (`bool`, *optional*, defaults to `model.config.output_scores` or `False` if the config does
+                not set any value): Whether or not to return the prediction scores. See `scores` under returned tensors
                 for more details.
-            output_scores (`bool`, *optional*, defaults to `False`):
-                Whether or not to return the prediction scores. See `scores` under returned tensors for more details.
-            return_dict_in_generate (`bool`, *optional*, defaults to `False`):
-                Whether or not to return a [`~utils.ModelOutput`] instead of a plain tuple.
-            forced_bos_token_id (`int`, *optional*):
+            return_dict_in_generate (`bool`, *optional*, defaults to `model.config.return_dict_in_generate` or `False`
+                if the config does not set any value): Whether or not to return a [`~utils.ModelOutput`] instead of a
```

Comment:
```suggestion
            return_dict_in_generate (`bool`, *optional*, defaults to `model.config.return_dict_in_generate` or `False` if the config does not set any value):
                Whether or not to return a [`~utils.ModelOutput`] instead of a
```